testing/Google.Api.Gax.Testing/FakeClock.cs

```diff
@@ -22,11 +22,6 @@ public class FakeClock : IClock
         private long _ticks;
 
         /// <summary>
-        /// Event raised (synchronously) whenever the clock is advanced.
-        /// </summary>
-        public event EventHandler<TimeChangedEventArgs> TimeChanged;
```

Comment:
Delete TimeChangedEventArgs? (below, GitHub won't let me add a comment where it's defined!)